internal/alert/interpolator.go

```diff
@@ -4,6 +4,9 @@ import (
 	"fmt"
 
 	"github.com/base-org/pessimism/internal/core"
+
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
```

Comment:
Why aren't we using `strings.ToUpper()` instead?

Reply:
strings.ToUpper makes the whole word uppercase and ToTitle does the same. The `strings` standard lib pkg seems to have deprecated `strings.Title`... if you look [here](https://pkg.go.dev/strings#Title) this is actually how they recommend doing title casing

Reply:
Fascinating